CHANGELOG.md

```diff
@@ -4,6 +4,59 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.0.0-rc1 (2020-10-06)
+
+This release adds major new features and bugfixes since the 1.7.4 release.
+We deem it moderate priority for upgrading.
+
+This release adds the long-awaited support for distributed hypertables to
+TimescaleDB. This release also adds support for user-defined actions allowing
+users to define actions that are run by the TimescaleDB automation framework.
```

Comment:
```suggestion
This release adds the long-awaited support for distributed hypertables to
TimescaleDB. With 2.0, users can run multiple instances of TimescaleDB,
configured so that one serves as an access node and multiple others as data
nodes, so that all queries for a distributed hypertable are issued to the access 
node, but inserted data and queries are pushed down across data nodes for
greater scale and performance.

This release also adds support for user-defined actions allowing
users to define actions that are run by the TimescaleDB automation framework.
```